README.md

```diff
@@ -0,0 +1,210 @@
+`versions` dictionary. Fill all the required information similar to the previous entries.
+For example, the value _v230913_ in the _versions_ dictionary describes the 13-Sep-2023 release.
+````python
+versions = {
+    "v230913": {
+        'date': '13-Sep-2023',
+        'ver_str_long': 'version 230913',
+        'ver_str_short': '230913',
+        "folder": "v230913",
+        "transpose": False,
+        "filename": "0. GCP-CEM.csv",
+        'ref': '10.5281/zenodo.8339353',
+        'ref2': '10.5194/essd-11-1675-2019',
+        'title': 'Global CO2 emissions from cement production',
+        'institution': "CICERO - Center for International Climate Research",
+        'filter_keep': {},
+        'filter_remove': {},
+        'contact': "johannes.guetschow@climate-resource.com",
+        'comment': ("Published by Robbie Andrew, converted to PRIMAP2 format by "
+                    "Johannes Gütschow"),
+        'unit': 'kt * CO2 / year',
+        'country_code': True,
+    },
+}
+````
+Then run the two commands `read_version` and `download_version` as described in [Contributing](#2-contributing). 
+
+## Help
+Show all doit commands
+```
+doit help
+```
+See a list with possible doit commands specific to this repository
+```
+doit list
+```
+
+Get help on a specific command
+
+```
+doit help <command>
+```
+
+## Contributing
+### Repository structure
+- **.datalad/** contains config file for datalad
+- **downloaded_data/** contains original data from Zenodo.
+- **extracted_data/** contains data in .csv and .nc format
+- **literature/** contains link to publication by Robbie M. Andrew. Can be downloaded with _datalad get_ command
+- **src/** 
+  - **download_version.py** downloads files from zenodo for a given version. The version to read will be taken from the command line using _argparse_.
+  - **download_version_datalad.py** calls datalad to run the data reading function.
+  - **helper_functions.py** contains a function to map country codes.
+  - **read_version.py** reads the data for a given version and saves to [PRIMAP2](https://primap2.readthedocs.io/en/stable/) native and
+    interchange format.
+  - **read_version_datalad.py** calls datalad to run the data reading function.
+  - **version.py** is a dictionary that contains metadata for each release. This file should be updated when [adding a new version](#a-namenewversiona-how-to-add-a-new-version) 
+- **dodo.py** defines pydoit commands.
+- **pyproject.toml** configuration file
+- **requirements.txt** requirements
+- **requirements_dev.txt** development requirements
+- **setup.cfg** requirements
+- **setup.py** installs python packages
+
+### Make sure to correctly set up the DataLad siblings
+Git repositories can configure clones of a dataset as _remotes_ in order to fetch, pull, or push from and to them. A `datalad sibling` is the equivalent of a git clone that is configured as a remote. 
```

Comment:
```suggestion
### Make sure to correctly set up the DataLad siblings

Git repositories can configure clones of a dataset as remotes in order to fetch, pull, or push from and to them. A `datalad sibling` is the equivalent of a git clone that is configured as a remote. 
```

Is this how datalad describes siblings? The language seemed a bit odd to me but maybe this is the best description